src/app/shared/components/color-picker/color-picker.component.scss

```diff
@@ -68,3 +72,8 @@ $input-margin: 10px;
   padding: 5px;
   box-sizing: content-box;
 }
+
+.color-picker-no-field {
+  margin-left: 47%;
```

Comment:
I understand percentage with margin left. but why did you use margin-top with percents?